src/ralph/conf.py

```diff
@@ -325,8 +324,10 @@ class Config(BaseSettingsConfig):
     RUNSERVER_POINT_IN_TIME_KEEP_ALIVE: str = "1m"
     RUNSERVER_PORT: int = 8100
     SENTRY_DSN: str = None
+    SENTRY_EXECUTION_ENVIRONMENT: str = "development"
     SENTRY_CLI_TRACES_SAMPLE_RATE = 1.0
-    SENTRY_LRS_TRACES_SAMPLE_RATE = 0.1
+    SENTRY_LRS_TRACES_SAMPLE_RATE = 1.0
+    SENTRY_IGNORE_HEALTH_CHECKS = False
```

Comment:
Everyone deserves typing :stuck_out_tongue: 

```suggestion
    SENTRY_LRS_TRACES_SAMPLE_RATE: float = 1.0
    SENTRY_IGNORE_HEALTH_CHECKS: bool = False
```